src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -146,6 +147,13 @@ mobile_apis::Result::eType CreateInteractionChoiceSetRequest::CheckChoiceSet(
       LOG4CXX_ERROR(logger_, "Incoming choice set has contains \t\n \\t \\n");
       return mobile_apis::Result::INVALID_DATA;
     }
+    for (choice_set_it_next = choice_set_it + 1;
+         choice_set->end() != choice_set_it_next;
+         ++choice_set_it_next) {
```

Comment:
@dcherniev, I still prefer not to change the variables names because _set  prefix indicates that we have a choiseSet and _it prefix indicates that we use an iterator.